### PR TITLE
New examples.

### PR DESCRIPTION
**Pull request by polhomarkho**
Some new examples for new and existing features:

- Sliders:
  - vertical
  - horizontal
  - drag / direct
- Midi
- SMS (send and receive)

Closes issues #117 and #113

### CODE REVIEW

File: PHONK-examples/Graphical User Interface/Sliders/main.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/*
+ *  Description tests every possible slider configuration.
```
Comment (victordiaz) [inline PHONK-examples/Graphical User Interface/Sliders/main.js:2]: Could you make the header so its like
https://github.com/victordiaz/PHONK/blob/develop/PHONK-examples/Device/Info/main.js

PHONK example: blbalablabla :)
you can add your name if you want :)

File: PHONK-examples/Graphical User Interface/Sliders/main.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ *  'direct' slider moves the slider to the position you clicked
+ *   while 'drag' slider moves only if you move you finger.
+ */
+
```
Comment (victordiaz) [inline PHONK-examples/Graphical User Interface/Sliders/main.js:6]: I would add the addTitle and addSubtitle, I think is good to show in the examples.

File: PHONK-examples/Graphical User Interface/Sliders/main.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+ui.addSlider(0.5, 0.75, 0.24, 0.24)
+  .range(0, 100)
+  .mode('direct')
+  .verticalMode(false);
+
+ui.addSlider(0.75, 0.75, 0.24, 0.24)
+  .range(0, 100)
+  .mode('drag')
+  .verticalMode(false);
```
Comment (victordiaz) [inline PHONK-examples/Graphical User Interface/Sliders/main.js:79]: I realized that some are duplicated. For sake of simplicity I would just show the 4 variants that we have now.

File: PHONK-examples/Network/SMS/main.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+/*
+ *  Description Send and receive a SMS. Requires the extended version of Phonk.
```
Comment (victordiaz) [inline PHONK-examples/Network/SMS/main.js:2]: add the `PHONK example: Name` header

File: PHONK-examples/Network/SMS/main.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+ui.addTitle(app.name);
+
```
Comment (victordiaz) [inline PHONK-examples/Network/SMS/main.js:6]: add subtitle

File: PHONK-examples/Peripherals/Low level midi to computer/main.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
+/*
+ *  Description simple midi controller to control a soft synth on a computer
```
Comment (victordiaz) [inline PHONK-examples/Peripherals/Low level midi to computer/main.js:2]: Add standard example header
and title and subtitle :)

**Review by victordiaz — COMMENTED** [review]
All good! There is some minor tweaks, it would be nice to add. Basically following the same standard example header and use addTitle and addSubtitle in each example. I think it helps to see which example you are running :)

**Comment by victordiaz** [comment]
Hey @polhomarkho

Do you mind if I do the changes I proposed and merge the pull request?
I want to do a release with the MIDI stuff you contributed soon.

**Comment by polhomarkho** [comment]
Hello @victordiaz, sorry for being so late. I didn't have much time these months but it's starting to calm down 😃 thank you for merging this!

**Pull request merged**